src/isPalindrome.js

```diff
@@ -0,0 +1,14 @@
+export default function isPalindrome(str) {
+  const formattedStr = str.toLowerCase().replace(/[^a-z|0-9]/g, "")
+  function reversedString(formattedStr) {
+
+    if (formattedStr === ""){
+      return "";
+    } else {
+      return reversedString(formattedStr.substr(1)) + formattedStr.charAt(0)
```

Comment:
implement a ternary? but that could be your preference if you want to be more explicit or not (keeping if else makes it more explicit)